app/views/local_authorities/index.html.erb

```diff
@@ -1,40 +1,36 @@
-<% content_for :page_title, "Local Authorities" %>
+<% content_for :page_title, "Councils" %>
 
 <% breadcrumb :local_authorities %>
 
-<div class="page-title">
-  <h1>Local Links Manager</h1>
-</div>
+<%= render "govuk_publishing_components/components/heading", {
+  text: "Councils (#{@authorities.count})",
+  margin_bottom: 5,
+} %>
 
-<%= render partial: 'shared/home_nav' %>
+<%
+  @table_rows = @authorities.map do |authority|
+    [
+      { text: link_to(authority.name, local_authority_path(authority.slug, filter: 'broken_links')) },
+      { text: authority.active? ? "Yes" : "No" },
```

Comment:
We can add the Design System link formatting `govuk-link` here:
```
{ text: link_to(authority.name, local_authority_path(authority.slug, filter: 'broken_links', class: 'govuk-link')) },
```

Reply:
Done!